pkg/ingester/ingester.go

```diff
@@ -821,6 +829,31 @@ func (i *Ingester) updateLoop(ctx context.Context) error {
 	}
 }
 
+func (i *Ingester) updateTSDBMaxExemplars() {
```

Comment:
Since we will support OOO config in the future, can we rename the function to not only `updateTSDBMaxExemplars`? Maybe something like `updateUserTSDBs` or `updateDBConfigs`, etc.

Reply:
How about `updateUserTSDBConfigs`?

Reply:
SGTM!